playground/nextjs/src/pages/render-notification/index.tsx

```diff
@@ -0,0 +1,24 @@
+import { novuConfig } from '@/utils/config';
+import { Inbox } from '@novu/react';
+
+export default function Home() {
+  return (
+    <div className="text-center flex flex-col gap-4">
+      <p className="text-xl">Render Notification Props</p>
+      <Inbox
+        options={novuConfig}
+        renderNotification={(notification) => {
+          return (
+            <div className="flex gap-2 flex-nowrap items-start self-stretch my-1 p-2 hover:bg-slate-200">
+              <div className="rounded-full w-8 h-8 overflow-hidden border border-cyan-200">Avatar</div>
+              <div>
+                <div className="text-xl font-bold">{notification.subject ?? 'Subject'}</div>
```

Comment:
```suggestion
                <div className="text-xl font-bold">{notification.subject || 'Subject'}</div>
```
Unless we want `''` to intentionally render an empty subject